docs, host-devices: Add SR-IOV vGPU preparation section (#1009)
kubevirt/user-guide: docs/compute/host-devices.md

## Preparation of PCI devices for passthrough

> At this time, KubeVirt is only able to assign PCI devices that are using the `vfio-pci` driver (except for [SR-IOV vGPU](#preparation-of-vgpu-devices-with-sr-iov)). To prepare a specific device for device assignment, it should first be unbound from its original driver and bound to the `vfio-pci` driver.

Reviewer: ```suggestion
At this time, KubeVirt is only able to assign PCI devices that are using the `vfio-pci` driver, with the exception of [SR-IOV vGPU](#preparation-of-vgpu-devices-with-sr-iov). To prepare a specific device for device assignment, it should first be unbound from its original driver and bound to the `vfio-pci` driver.
```

Once the mdev is configured, KubeVirt will be able to discover and use it for device assignment.

## Preparation of vGPU devices with SR-IOV

In general, configuration of vGPUs with SR-IOV should be done according to the vendor directions.

> KubeVirt doesn't support creation of virtual function (VF) nor the assignment of vGPU profile to the function.

Reviewer: ```suggestion
KubeVirt does not support the creation of virtual function (VF) nor the assignment of vGPU profile to the function.
```

> KubeVirt only facilitates exposure of these virtual functions for the VM to consume them (through generic PCI device plugin) but doesn't have fine-grained distinctions of the profiles such as for the mdevs. Therefore it is recommended to use vendor's device plugin to expose the functions in case of heterogeneous profiles.

Reviewer: ```suggestion
KubeVirt only facilitates exposure of these VFs for the VM to consume them (through generic PCI device plugin) but does not have fine-grained distinctions of the profiles, like those for mdevs. Therefore it is recommended to use vendor's device plugin to expose the functions in case of heterogeneous profiles.
```
?
